docker-compose.yml

```diff
@@ -51,6 +51,7 @@ services:
       SPRING_CLOUD_CONFIG_LABEL: "main"
       SPRING_CLOUD_CONFIG_URI: "http://config-server:8888"
       LOGGING_LEVEL_ROOT: ${LOGGING_LEVEL_ROOT:-INFO}
+      ROLE_SERVICE_URI: "http://role-service:8082"
```

Comment:
I add the following to security application.yml - thats where the other services xx.service.uri details are..

role:
  service:
    name: role-service
    uri: http://role-service
    endpoint:
      sites: /roles

Reply:
> I add the following to security application.yml - thats where the other services xx.service.uri details are..
> 
> role:
> service:
> name: role-service
> uri: http://role-service
> endpoint:
> sites: /roles

Good call thanks Darren - I've moved this to application.yaml with the others.